<?xml version="1.0"?>
<launch>
  <arg name="num_samples" default="1000" />
  <arg name="chain_start" default="base_link" />
  <arg name="chain_end" default="wrist_3_link" />
  <arg name="timeout" default="0.05" />

  <param name="robot_description" command="$(find xacro)/xacro.py '$(find ur5_with_sr300_support)/urdf/ur5_robot.urdf.xacro'" />


  <node name="trac_ik_test" pkg="plan_pkg" type="tSim" output="screen">
    <param name="num_samples" value="$(arg num_samples)"/>
    <param name="chain_start" value="$(arg chain_start)"/>
    <param name="chain_end" value="$(arg chain_end)"/>
    <param name="timeout" value="$(arg timeout)"/>
    <param name="urdf_param" value="/robot_description"/>
  </node>


</launch>
